README.md

```diff
@@ -395,7 +388,7 @@ rails g react:component Post title:string published:bool published_by:instanceOf
 would generate:
 
 ```js
-var Post = React.createClass({
+class Post extends React.Component{
```

Comment:
If we change the documentation for ComponentGenerator, should we also update the code to match the new documentation?